example/http/client/coro/http_client_coro.cpp

```diff
@@ -52,15 +54,18 @@ do_session(
 
     // These objects perform our I/O
     tcp::resolver resolver{ioc};
-    tcp::socket socket{ioc};
+    stream_type stream(ioc);
```

Comment:
The inconsistency of initialization methods might be confusing to some readers.

Reply:
Yeah I need to go through all the examples and clean that up